File: JAVA/Day02/QueueExample.java
import java.util.*;

public class QueueExample {
    public static void main(String[] args) {
        Queue<Integer> queue = new LinkedList<>();
        
        queue.offer(10);
        queue.offer(20);
        queue.offer(30);
        
        System.out.println("Queue: " + queue);
        
        int removedElement = queue.poll();
        System.out.println("Removed Element: " + removedElement);
        
        int frontElement = queue.peek();
        System.out.println("Front Element: " + frontElement);
        
        boolean isEmpty = queue.isEmpty();
        System.out.println("Is Queue Empty? " + isEmpty);
    }
}

File: C#/Day02/QueueExample.cs
using System;
using System.Collections.Generic;

public class QueueExample {
    public static void Main(string[] args) {
        Queue<int> queue = new Queue<int>();
        
        queue.Enqueue(10);
        queue.Enqueue(20);
        queue.Enqueue(30);
         
        Console.WriteLine("Queue: " + string.Join(", ", queue));
        
        int removedElement = queue.Dequeue();
        Console.WriteLine("Removed Element: " + removedElement);
        
        int frontElement = queue.Peek();
        Console.WriteLine("Front Element: " + frontElement);
        
        bool isEmpty = queue.Count == 0;
        Console.WriteLine("Is Queue Empty? " + isEmpty);
    }
}
